### PR TITLE
feat: improve error messages

### PR DESCRIPTION
**Pull request by odsod**

### CODE REVIEW

File: reflect/aipreflect/validateresourcereferences_test.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func TestValidateResourceReferences(t *testing.T) {
 				OriginSite:      "shippers/1",
 				DestinationSite: "shippers/1/sites/2",
 			},
-			errorContains: "value shippers/1 of field origin_site is not a valid resource reference",
+			errorContains: "value 'shippers/1' of field origin_site is not a valid resource reference",
```
Comment (ericwenn) [inline reflect/aipreflect/validateresourcereferences_test.go:52]: include the rest of error message here

File: reflect/aipreflect/validateresourcereferences.go [hunk 2]
```diff
@@ -59,7 +59,7 @@ func ValidateResourceReferences(message proto.Message) error {
 					}
 				}
 				errValidate = fmt.Errorf(
-					"value %s of field %s is not a valid resource reference for %s",
+					"value '%s' of field %s is not a valid resource reference for %s",
```
Comment (ericwenn) [inline reflect/aipreflect/validateresourcereferences.go:62]: How does the last verb look?

Comment (odsod) [reply reflect/aipreflect/validateresourcereferences.go:62]: updated

File: reflect/aipreflect/validateresourcereferences_test.go (hunk 1 above)
Comment (odsod) [reply reflect/aipreflect/validateresourcereferences_test.go]: updated

**Review by ericwenn — APPROVED** [review]

**Pull request merged**